<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QListWidget" name="layerList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QFrame" name="mapFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutMap">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddLayer"/>
   <addaction name="actionAddPoint"/>
   <addaction name="actionSaveMap"/>
   <addaction name="separator"/>
   <addaction name="actionZoom"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionShapeInfo"/>
  </widget>
  <action name="actionAddPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/addpoint.png</normaloff>:/images/addpoint.png</iconset>
   </property>
   <property name="text">
    <string>Add point</string>
   </property>
   <property name="toolTip">
    <string>Add a point to current layer</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to layer extent</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionAddLayer">
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/addlayer.png</normaloff>:/images/addlayer.png</iconset>
   </property>
   <property name="text">
    <string>Add layer</string>
   </property>
   <property name="toolTip">
    <string>Add new empty layer</string>
   </property>
  </action>
  <action name="actionSaveMap">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/savemap.png</normaloff>:/images/savemap.png</iconset>
   </property>
   <property name="text">
    <string>Save map</string>
   </property>
   <property name="toolTip">
    <string>Save current layer</string>
   </property>
  </action>
  <action name="actionShapeInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="map.qrc">
     <normaloff>:/images/mapinfo.png</normaloff>:/images/mapinfo.png</iconset>
   </property>
   <property name="text">
    <string>ShapeInfo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="map.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionAddPoint</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>addPointTriggered(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveMap</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveLayer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerList</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>layerListChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>layerList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>layerListSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShapeInfo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>shapeInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addPointTriggered(bool)</slot>
  <slot>zoom()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>addLayer()</slot>
  <slot>saveLayer()</slot>
  <slot>layerListChanged(QListWidgetItem*)</slot>
  <slot>layerListSelected()</slot>
  <slot>shapeInfo()</slot>
 </slots>
</ui>
